mucommander-viewer-text/src/main/java/com/mucommander/viewer/text/TextEditorImpl.java

```diff
@@ -199,7 +199,8 @@ public void mouseWheelMoved(MouseWheelEvent e) {
     }
 
     void find() {
-        FindDialog findDialog = new FindDialog(frame);
+        boolean isEditable = textArea.isEditable();
```

Comment:
for consistency, the variable shouldn't start with 'is'
```suggestion
        boolean editable = textArea.isEditable();
```